libs/estdlib/src/binary.erl

```diff
@@ -115,3 +117,37 @@ split(_Binary, _Pattern) ->
 -spec split(Binary :: binary(), Pattern :: binary(), Option :: [global]) -> [binary()].
 split(_Binary, _Pattern, _Option) ->
     erlang:nif_error(undefined).
+
+%%-----------------------------------------------------------------------------
+%% @equiv replace(Binary, Pattern, Replacement, [])
+%% @param   Binary      binary where the replacements should occur
+%% @param   Pattern     binary pattern which is to be replaced
+%% @param   Replacement binary which will replace the pattern
+%% @return  resulting binary after replacements
+%% @doc     Replaces first occurrence of Pattern in Binary with Replacement.
+%%          If Pattern is not found, returns the original binary unchanged.
+%%          Pattern and Replacement must be binaries.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec replace(Binary :: binary(), Pattern :: binary(), Replacement :: binary()) -> binary().
+replace(_Binary, _Pattern, _Replacement) ->
+    erlang:nif_error(undefined).
+
+%%-----------------------------------------------------------------------------
+%% @param   Binary      binary where the replacements should occur
+%% @param   Pattern     binary pattern which is to be replaced
+%% @param   Replacement binary which will replace the pattern
+%% @param   Options     list of options for the replacement operations.
+%% @return  resulting binary after replacements
+%% @doc     Replaces occurrences of Pattern in Binary with Replacement.
+%%          If Options includes `global`, replaces all occurrences; otherwise, replaces just the first occurrence.
+%%          If Pattern is not found, returns the original binary unchanged.
+%%          Pattern and Replacement must be binaries.
+%%          Only implemented option is `global`.
```

Comment:
we have a tricky documentation system, quotes must be like `foo' otherwise CI will fail.